tests/check/signed_video_helpers.h

```diff
@@ -73,43 +73,44 @@ get_initialized_signed_video(SignedVideoCodec codec,
 test_stream_t *
 create_signed_nalus(const char *str, struct sv_setting settings);
 
-/* See function create_signed_nalus_int, with the diffrence that each NALU is split in two parts. */
+/* See function create_signed_nalus_int, with the diffrence that each NAL Unit is split in
```

Comment:
nit: diffrence->difference. (It's only testcode so skip it if you want)

Reply:
I've made the change in a later patch to simplify rebase work